crates/core/src/client/config.rs

```diff
@@ -0,0 +1,37 @@
+use eyre::{eyre, Result};
+use starknet::core::types::FieldElement;
+
+fn get_env_var(name: &str) -> Result<String> {
```

Comment:
Based on @greged93 's former PRs we could use `thiserror` here to define a configerror, and then add it in the more general error enum as a transparent error